salt/modules/ddns.py

```diff
@@ -75,8 +75,8 @@ def _get_keyring(keyfile):
     return keyring
 
 
-def add_host(zone, name, ttl, ip, nameserver='127.0.0.1', replace=True,
-             timeout=5, **kwargs):
+def add_host(zone, name, ttl, ip, nameserver='127.0.0.1', port=53,
```

Comment:
Could you add the new argument after the other established kwargs? There are people who call these using positional arguments and this will break for them.